ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/fragmentisers/geospatial/constants/GeospatialConstants.java

```diff
@@ -0,0 +1,12 @@
+package be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.constants;
+
+public class GeospatialConstants {
+    private GeospatialConstants() {
+    }
+
+    public static final String FRAGMENT_KEY_TILE = "tile";
+
+    public static final String GEOSPARQL_AS_WKT = "http://www.opengis.net/ont/geosparql#asWKT";
```

Comment:
this should come from the config, so where is this used?

Reply:
created an issue for this: https://github.com/Informatievlaanderen/VSDS-LDESServer4J/issues/160